pages/apim/3.x/installation-guide/docker/installation-guide-docker-customize.adoc

```diff
@@ -37,30 +51,47 @@ This requires a couple of additional environment parameters on starting the gate
   --env gravitee_ratelimit_redis_port=6379 \  
 ----
 
-and you can then drop the equivalent parameter for mongodb
+You also need to remove the `gravitee_ratelimit_mongodb_uri` `env` from the command.
+
+[NOTE]
+====
+Your Redis host and port may be different.
+====
+
+== Use JDBC connection as the datastore for management
```

Comment:
For JDBC, the plugin is already included in the bundle. However, it requires downloading a driver manually. We have a documentation there: https://docs.gravitee.io/apim/3.x/apim_installguide_repositories_jdbc.html

Reply:
I have updated the text and added a link to that page. I also made a small change to that page. It will need reviewed too.